src/pages/dashboard/OperatorHbarChart/OperatorHbarChart.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import './operatorsHbarChart.scss'
+
+const colorsByCompannies: { [index: string]: string } = {
+  'אגד תעבורה': 'rgb(47, 146, 80)',
+  אגד: 'rgba(0, 132, 121)',
+  'אלקטרה אפיקים': '#9ACA3C',
+  קווים: '#03296A',
+  סופרבוס: '#164C8F',
+  דן: '#205CAB',
+  'נתיב אקספרס': '#F2BD00',
+  מטרופולין: '#FF8500',
+}
+
+function getColorByHashString(str: string) {
+  let hash = 0
+  for (let i = 0; i < str.length; i++) {
+    hash = str.charCodeAt(i) + ((hash << 5) - hash)
+  }
+  let color = '#'
+  for (let i = 0; i < 3; i++) {
+    const value = (hash >> (i * 8)) & 0xff
+    color += ('00' + value.toString(16)).substr(-2)
+  }
+  return color
+}
+
+function getColorName(name: string) {
+  return colorsByCompannies[name] || getColorByHashString(name)
+}
+
+function OperatorHbarChart({
+  operators,
+}: {
+  operators: { name: string; total: number; actual: number }[]
+}) {
+  const percents = operators.map((o) => (o.actual / o.total) * 100)
```

Comment:
careful of total 0

Reply:
`1234/0 * 100 + "%"` gives 'Infinity%'
that doesn't sounds wrong

Reply:
you can integrate it to `The bus is expected to arrive at ${1234/0} minutes`, will be pretty accurate :)

Reply:
think about it - the bus company didn't plan to drive a single bus... yet, somehow, they have managed to get busses driving. their success rates are infinity by definition 😂🤣